packages/financial-templates-lib/package.json

```diff
@@ -11,6 +11,7 @@
     "@uniswap/sdk": "^2.0.5",
     "bluebird": "^3.7.2",
     "dotenv": "^6.2.0",
+    "mathjs": "^9.2.0",
```

Comment:
As a general comment about smart contract interface development patterns, and not this pull request specifically, I wonder if it would be better for interfaces that are mostly reading and writing data from smart contracts to do most of their calculation (and maybe rendering) on the server side.

This came up in the engineering team discussion, where we determined that asking people who use this package to run a server would be a heavy lift, versus doing calculations in the browser (which `mathjs` allows for) or hitting an API we provide, which would be a centralized point of failure in addition to taking a lot of time to build and maintain.

In the long run, I think that smart contract front-ends that pull most of the work out of the client and put it on the server will win out, since very little of what you want to do with smart contracts resembles the social media clients that React _et al._ were really designed for. Every user interaction takes at least a few seconds to finalize, and data updates also take a few seconds. Doing most of the stuff on the server side with a light client-side interface feels like the right trade-off.

I suppose this is also some background for the general community on why we're going in this direction, since `mathjs`, while quite heavy, will work in the browser, while some other alternatives we considered would not.

Reply:
Yeah, I could imagine employing a much heavier server-side implementation. If we really wanted our UIs to be resilient, they could have a fallback setup where they _only_ pull down the price feed js code if the backend/server-side is down for some reason. This, while complicated, could be the best of both worlds, although I doubt we'll ever have the engineering resources to build something like that :)